superdesk.config.js

```diff
@@ -31,5 +31,20 @@ module.exports = function(grunt) {
             DEFAULT_SCHEDULE: {minutes: 5, seconds: 0},
             DEFAULT_IDLE_TIME: {hours: 0, minutes: 0},
         },
+        apps: [
```

Comment:
hey @fritzSF just exclude this configuration from superdesk.config.js

Reply:
Done